packages/core/src/state/reducer.ts

```diff
@@ -55,41 +60,37 @@ export default function reducer(
     }
     case RECEIVE_TYPE: {
       if (action.error) {
+        return reduceError(state, action, action.payload);
+      }
+      try {
```

Comment:
diff here is very weird (showed sensible things in vscode).

this is actually just putting everthing in try/catch

above we extracted the error handling case so we could reuse it when this throws